changelog.md

```diff
@@ -9,6 +9,10 @@
   - Introduced a new `accessoryView` prop that enables rendering any view on the leading side of the text input.
 - react-native-bpk-component-phone-input:
  - New `BpkDiallingCodeList` component.
+- bpk-component-text:
+  - Svg's fill colour is automatically set to text colour.
```

Comment:
I think this could be more descriptive. How about:

```
- SVGs inside `BpkText` will now automatically have their fill colour set to match the text colour.
```

Same for below except for `BpkLink`.